Plugins/Gerrit/FormGerritPublish.Designer.cs

```diff
@@ -129,15 +130,35 @@ private void InitializeComponent()
             | System.Windows.Forms.AnchorStyles.Right)));
             this._NO_TRANSLATE_Topic.Location = new System.Drawing.Point(101, 78);
             this._NO_TRANSLATE_Topic.Name = "_NO_TRANSLATE_Topic";
-            this._NO_TRANSLATE_Topic.Size = new System.Drawing.Size(522, 23);
+            this._NO_TRANSLATE_Topic.Size = new System.Drawing.Size(522, 21);
             this._NO_TRANSLATE_Topic.TabIndex = 6;
             // 
+            // _NO_TRANSLATE_Reviewers
+            // 
+            this._NO_TRANSLATE_Reviewers.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
+            | System.Windows.Forms.AnchorStyles.Right)));
+            this._NO_TRANSLATE_Reviewers.Location = new System.Drawing.Point(101, 108);
+            this._NO_TRANSLATE_Reviewers.Name = "_NO_TRANSLATE_Reviewers";
+            this._NO_TRANSLATE_Reviewers.Size = new System.Drawing.Size(522, 21);
+            this._NO_TRANSLATE_Reviewers.TabIndex = 11;
+            // 
+            // label1
+            // 
+            this.label1.AutoSize = true;
+            this.label1.Location = new System.Drawing.Point(13, 111);
+            this.label1.Name = "label1";
+            this.label1.Size = new System.Drawing.Size(69, 13);
+            this.label1.TabIndex = 10;
```

Comment:
Please make sure the tab order on the form is meaningful. Adjust if necessary

Reply:
Fixed